tasks/src/document-question-answering/about.md

```diff
@@ -0,0 +1,45 @@
+## Use Cases
+
+Document question answering models can be used to answer natural language questions directly from document images. This is useful when the visual information of the document is important for answering the question.
+
+### Document Parsing
+
+You can use Document question answering models to parse forms. For example, you can extract the name, address, and other information from a form. You can also use the model to extract information from a table, or even a resume.
+
+### Invoice Information Extraction
+
+You can use Document question answering models to extract information from invoices. For example, you can extract the invoice number, the invoice date, the total amount, the VAT number, and the invoice recipient.
```

Comment:
Might want to provide some detail about why this is called out, e.g. "One of the most popular use cases is ... "